pages/models/generations/GENTPJ/index.md

```diff
@@ -0,0 +1,233 @@
+tags: ["#196","Synchronous machine", "Rotor", "Stator", "PowerWorld", "Opensource", "GENROU", "GENSAL", "GENTPF", "GENTPJ", "Saturation", ]
+date: 05/08/2024 
+last-updated: 07/08/2024
+id: #196
+authors: Carlos Alegre (eRoots)
+reviewers: Eduardo Prieto Araujo (UPC), Josep Fanals Batllori (eRoots)
+---
+
+## Context
+
+Saturation modelling in synchronous machines is key to perform transient stability studies. Starting from the dynamic model of the Synchronous Machine, [GENSAL/GENROU without saturation](../GENSAL/) [[1]](#1), some existing models that incorporate the saturation of the poles have been developed. There are some models that directly incorporate the saturation with some additive corrections in the dynamic equation, such as the GENROU/GENSAL with saturation, but they are somewhat obsolete [[2]](#2), and mismatching between the measured field current and voltage measurements with the values predicted have been found. The GENTPF/GENTPJ models [[3]](#3) aimed to apply the saturation effects to the parameters of the machine directly using a scaling factor.
+
+## Model use, assumptions, validity domain and limitations
+
+The model is used to perform transient stability analysis of synchronous machines, providing a more accurate representation of the physical machine as it considers that there exist saturation in the iron of the machine, and it is applied to all the inductive terms simultaneously, differently to what the GENROU/GENSAL models with additive saturation consider. Following a recommendation note from the North-America Electrical Reliability Corporation (NERC) [[4]](4), the GENTPJ is overall recommended for all the synchronous machines as it recognizes the effect of the stator current in saturation. The GENROU, GENSAL and GENTPF (the later being exactly identical to the GENTPJ except an additive term in the saturation function) may induce significant error specially in simulations where reactive power support is considered.
+
+The model assumptions are the following:
+
+* The three-phase system is balanced
+* The air gap between the stator and rotor is non-uniform, but the relationships between the flux linkages and currents are independent of the position of the rotor when expressed in the *dq0* reference frame.
+* The magnetic circuit is considered to be non-linear, considering saturation effects, and can be modeled using different saturation functions.
+* The saturation effects are applied to the subtransient reactances.
+
+The GENTPJ model is suitable for EMT as it models with detail the non-linearities and the fast dynamics of the machine. However, the saturation effects are only applied to the subtransient reactances. Newer models consider saturation over even more elements of the machine, improving GENTPF and GENTPJ models. GENTPJ will be deprecated in December 2024, and substituted by the [GENQEC](../GENQEC/) [[5]](#5), although this later model is still in testing process [[6]](#6).
```

Comment:
[GENQEC](../GENQEC/) - link is not working

Reply:
This one will work once the GENQEC model is uploaded